server/pulp/server/db/manage.py

```diff
@@ -116,6 +116,7 @@ def ensure_database_indexes():
     collection does not already exist.
     """
     model.RepositoryContentUnit.ensure_indexes()
+    model.Repository.ensure_indexes()
```

Comment:
please add the testcase for this like here https://github.com/pulp/pulp/blob/master/server/test/unit/server/db/test_manage.py#L118